haiku-apps/trackgit/trackgit-1.0.0~git.recipe

```diff
@@ -2,31 +2,31 @@ SUMMARY="Tracker add-on for git"
 DESCRIPTION="Tracker Addon for Git Version Control System. \
 TrackGit provides user with GUIs for various Git commands. The implementation \
 was done using LibGit2. TrackGit uses the Dynamic population of menu."
-HOMEPAGE="https://github.com/Hrily/TrackGit"
+HOMEPAGE="https://github.com/HaikuArchives/TrackGit"
 COPYRIGHT="2018 Hrishi Hiraskar
 	2018 HaikuArchives Team"
 LICENSE="MIT"
-REVISION="1"
-srcGitRev="cce7abfecd0dcaf1ed0106a9adb6ce853393f697"
-SOURCE_URI="https://github.com/Hrily/TrackGit/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="f8ba2f9f6ce6c56d88d4dabaa14a24112599a3ec4b643518d3e4230fb628d1dc"
+REVISION="2"
+srcGitRev="e38a8e78b4a1f3c5dd6431b4fe2d07e8668227d0"
+SOURCE_URI="https://github.com/HaikuArchives/TrackGit/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="64ffad6aad5b81f3b1445f61701138d0808d6144ed8f753906151a3d412ccaaa"
 SOURCE_FILENAME="trackgit-$srcGitRev.tar.gz"
 SOURCE_DIR="TrackGit-$srcGitRev"
 
-ARCHITECTURES="x86_gcc2 !x86 !x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
 
 PROVIDES="
 	trackgit = $portVersion
 	app:TrackGit = $portVersion
 	"
 REQUIRES="
 	haiku
-	lib:libgit2
+	lib:libgit2 >= 0.27
```

Comment:
```suggestion
	lib:libgit2
```

You don't have to specify the version constraint here, as HaikuPorter will derive it from the build environment automatically